tests/async/test_defaultbrowsercontext.py

```diff
@@ -57,7 +57,7 @@ async def test_context_cookies_should_work(server, launch_persistent):
             "expires": -1,
             "httpOnly": False,
             "secure": False,
-            "sameSite": "None",
+            "sameSite": "Lax" if is_firefox else "None",
```

Comment:
Why did this change?

Reply:
Andrey says that Firefox has switched the defaults.

Reply:
"There is currently no timeline to ship this feature to the release channel of Firefox.". Let's make sure to fix it upstream. Filing a bug